ledger-service/http-json/src/main/scala/com/digitalasset/http/CommandService.scala

```diff
@@ -68,15 +69,24 @@ class CommandService(
       input: CreateCommand[lav1.value.Record, TemplateId.RequiredPkg],
   )(implicit
       lc: LoggingContextOf[InstanceUUID with RequestID]
-  ): Future[Error \/ ActiveContract[lav1.value.Value]] =
+  ): Future[Error \/ domain.CreateCommandResponse[lav1.value.Value]] =
```

Comment:
Minor: import domain.CreateCommandResponse ?

Reply:
Most uses of `domain` don't import its symbols directly, so it's always clear that you're dealing with the req/resp domain model at use site.